custom/icds_reports/static/js/directives/program-summary/program-summary.directive.js

```diff
@@ -14,7 +14,6 @@ function ProgramSummaryController($scope, $http, $log, $routeParams, $location,
     vm.isAlertActive = isAlertActive;
 
     vm.prevDay = moment().subtract(1, 'days').format('Do MMMM, YYYY');
-    vm.currentMonth = moment().format("MMMM");
```

Comment:
I didn't track down the commit where this dependency was removed, but was pretty careful about confirming it wasn't necessary (and leaving this variable in felt confusing)